lib/test_prof/vernier.rb

```diff
@@ -19,14 +19,15 @@ module TestProf
   module Vernier
     # Vernier configuration
     class Configuration
-      attr_accessor :mode, :target, :interval
+      attr_accessor :mode, :target, :interval, :hooks
 
       def initialize
         @mode = ENV.fetch("TEST_VERNIER_MODE", :wall).to_sym
         @target = (ENV["TEST_VERNIER"] == "boot") ? :boot : :suite
 
         sample_interval = ENV["TEST_VERNIER_INTERVAL"].to_i
         @interval = (sample_interval > 0) ? sample_interval : nil
+        @hooks = (ENV["TEST_VERNIER_HOOKS"] == "rails") ? :rails : nil
```

Comment:
We should probably take into account possible future named hooks; so, having smith like: `ENV["TEST_VERNIER_HOOKS"]&.split(",").map(&:to_sym)` would be better.

Reply:
done